0023-low-level-sampler.md

```diff
@@ -0,0 +1,188 @@
+# Low level sampler
+
+| **Status**        | **Proposed/Accepted/Deprecated** |
+|:------------------|:---------------------------------------------|
+| **RFC #**         | 0023                                         |
+| **Authors**       | Salva de la Puente (salva@ibm.com),  Antonio Corcole, Ian Hincks    |
+| **Deprecates**    | TBD                 |
+| **Submitted**     | 2025-07-30                                   |
+| **Updated**       | YYYY-MM-DD                                   |
+
+## Summary
+In Qiskit, the quantum computer interface is built around primitives, which are
+the fundamental operations executable on a quantum circuit. Currently, Qiskit
+offers two such primitives: the Sampler and the Estimator. The Estimator
+leverages advanced error mitigation techniques to enhance the quality of
+results from noisy quantum computers. On a noiseless quantum computer, the
+Estimator could instead be constructed atop the Sampler, simplifying its role.
```

Comment:
The purpose of the Estimator extends beyond just mitigating noise. It also adds a layer abstraction around the concept of estimating observables, allowing implementors of Estimator interfaces to offer different methods for observable estimation (e.g. classical shadows) while preserving a common interface.